### DISCUSSION TITLE
I would like to add the path on top in my changes.

### DISCUSSION THREAD

**Discussion by diegosrp** [cotes2020/jekyll-theme-chirpy#2112] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I would like to:
1. direct the homepage to the blog tab I created.
2. the homepage will be the content of my About tab, but I want the blog tab to be below the category tab.
3. remove the "about" title from the page, which I only managed to do by changing the layout to default. 
4. blog contains path on top.

I am not sure if is the best way, how I created:
Created a folder blog and moved the **index.html** inside.
```
---
layout: home
# Index page
---
```

**config.yml**
```
permalink: /blog/:title/
paginate_path: "/blog/page:num/"
```

**about.md**
```
---
layout: default
permalink: /
order: 4
---
```

**blog.md**
```
---
permalink: /blog
order: 2
---
```

**sidebar.html**
```
      <!-- home -->
      <li class="nav-item{% if page.url == '/' | relative_url %}{{ " active" }}{% endif %}">
        <a href="{{ '/' | relative_url }}" class="nav-link">
            <i class="fa-fw "></i>
            <span>{{ site.data.locales[include.lang].tabs.home | upcase }}</span>
        </a>
        </li>

      <!-- the real tabs -->
      {% for tab in site.tabs %}
      {% if tab.url == '/' | relative_url %}{% continue %}{% endif %}
        <li class="nav-item{% if tab.url == page.url %}{{ " active" }}{% endif %}">
          <a href="{{ tab.url | relative_url }}" class="nav-link">
            <i class="fa-fw {{ tab.icon }}"></i>
            {% capture tab_name %}{{ tab.url | split: '/' }}{% endcapture %}

            <span>{{ site.data.locales[include.lang].tabs.[tab_name] | default: tab.title | upcase }}</span>
          </a>
        </li>
```

It seems ok, but I noticed that at the top there is no page path like all the tabs, it looks like it was home, you can check that i write in red color "blog ??" because only show "home"
![Screenshot 2024-12-11 at 22 01 07](https://github.com/user-attachments/assets/2500af65-730a-44c1-9811-9dac276b8e06)
![Screenshot 2024-12-11 at 22 01 47](https://github.com/user-attachments/assets/d047eaea-e051-42af-9d31-28feb1a939b0)

summary: The changes make sense? Can I add the path to blog?

### Operations you have already tried

I tried to create it via the sidebar, but it is below the home page, and I couldn't order it like I can with the tabs.

### Anything else?

reference: https://jinchaoli.com/blog/exchange-homepage-and-about/
and other PR on the repo: https://github.com/cotes2020/jekyll-theme-chirpy

**Comment by SoumyaK4** [comment]
I suggested the following in #2099
You should be able to change the code a bit to get your desired results @diegosrp 

Please refer to this [demo repo](https://github.com/SoumyaK4/demo) I made
[Live demo](https://soumyak4.in/demo/)

## Changes

1. [Index.html](https://github.com/SoumyaK4/demo/blob/main/index.html)
2. The [static page](https://github.com/SoumyaK4/demo/blob/main/cv.md) you wanted
3. [Original sidebar](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/_includes/sidebar.html) vs [Modified sidebar](https://github.com/SoumyaK4/demo/blob/main/_includes/sidebar.html)
Create `_includes/sidebar.html` 
Changes at line 22 and additional lines at line 28
4. [Original Topbar](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/_includes/topbar.html) vs [Modified topbar](https://github.com/SoumyaK4/demo/blob/main/_includes/topbar.html)
Create `_includes/topbar.html`
Changes at line 11
5. New layout [for static page](https://github.com/SoumyaK4/demo/blob/main/_layouts/cv.html)
Create `_layouts/cv.html`

Still some minor changes might be necessary, if you can't figure them out let me know I'll share them whenever I get some time.
And remember, because there's code customisation involved, some things could break when you upgrade chirpy version, so keep a note of the changes you're making for future reference

**Reply by diegosrp** [reply]
Hi @SoumyaK4 , thanks for your feedback, I had already seen your post and saw that the path was correct, but I couldn't change the order of the menu, could you tell me how it can be done?
<img width="159" alt="Screenshot 2024-12-12 at 10 27 12" src="https://github.com/user-attachments/assets/68652c3a-5ca5-474b-a6a9-a345ebebb9e2" />

**Reply by SoumyaK4** [reply]
@diegosrp go to `_tabs`
Then `categories` and you'll see a line with `order`
That decides the order in the menu usually.
But in this custom case, either you create a blog tab in the tabs folder, or go to `_includes/sidebar.html` to remove the for loop and use if...else to order the menu...

If it's still hard to figure out let me know I'll modify and give the full code in a few minutes

**Reply by diegosrp** [reply]
Hello @SoumyaK4 ,

The screenshot I sent above shows the changes I made to the tabs. However, it does not include the path on top, as shown in the screenshot where "blog," for example, is missing.

If I follow your suggestion of including "blog" inside sidebar it works, but the issue with the order persists. Since "home" and "blog" are not in the tabs through the sidebar, I can't arrange them like the others, and I couldn't make this adjustment.

I appreciate your help !

**Reply by SoumyaK4** [reply]
@diegosrp I'll change stuff in my demo repo and ping you in a few hours...
Also if the problem is Topbar then `_includes/topbar.html` needs some changes probably 
Can you tell me exactly what order you need and what other changes you want please.
If possible draw a rough sketch and post here

**Reply by SoumyaK4** [reply, marked as answer]
@diegosrp check [these changes](https://github.com/SoumyaK4/demo/commit/642d4fec336eace2d9572b3cbc8e205855d2353f)

**Comment by diegosrp** [comment]
@SoumyaK4 it worked, thank you so much!

**Reply by SoumyaK4** [reply]
@diegosrp glad to know, do select the comment as answer so it helps other people as well

**Reply by diegosrp** [reply]
hey @SoumyaK4 how can I add reaction and comments on the about page? 

I saw that it is possible to change the layout for the post, but I would not like to use the layout as a whole because I only wanted to add the reaction and comments.

do you know how can I modified? I tried check the layout post but I can't identify relation with comments or reaction

**Reply by SoumyaK4** [reply]
@diegosrp that's only available for the post layouts. The default about layout is page.
You have to set about tab's layout as post to get comments.
or if you want the about page to look the same only adding comments and reactions, then that requires you to create a custom layout,
_**Edit:**_
but there's a lot of code overriding things I don't understand, I tried it but something is overriding the custom layout I'm trying to set because about is a tab, so my answer to this is a bit longer now:
move about.md to the root of your repo, then set the custom layout at about's frontmatter, then manually add the link and icon from about in `_includes/sidebar.html`

**Reply by diegosrp** [reply]
hmm..
I think I got it

I updated it as follows:

about.html
```
---
layout: default
script_includes:
  - comment
---
```

about.md
```
---
layout: about
permalink: /
comments: true
---
```


thanks @SoumyaK4  !

**Reply by SoumyaK4** [reply]
@diegosrp yup that's exactly what I meant <3
